google/cloud/aiplatform/training_jobs.py

```diff
@@ -1793,6 +1793,7 @@ def run(
         window_stride_length: Optional[int] = None,
         window_max_count: Optional[int] = None,
         holiday_regions: Optional[List[str]] = None,
+        enable_probabilistic_inference: Optional[bool] = None,
```

Comment:
If I'm reading the argument description correctly, it seems be either enabled or disabled. It may make more sense to default to `False` unless `None` has a different meaning than disabled.